.github/workflows/config-safety-check.yml

```
@@ -0,0 +1,103 @@
+      - 'tsconfig.json'
+      - '.github/workflows/*.yml'
+      - '*.config.js'
+      - '*.config.ts'
+
+jobs:
+  safety-check:
+    name: Validate Configuration Changes
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)